Content.Shared/Chemistry/ReagentUnit.cs

```diff
@@ -0,0 +1,184 @@
+﻿using Robust.Shared.Interfaces.Serialization;
+using Robust.Shared.Serialization;
+using System;
+using System.Linq;
+
+namespace Content.Shared.Chemistry
+{
+    [Serializable]
+    public struct ReagentUnit : ISelfSerialize
```

Comment:
You may want to implement some common interfaces like `IEquatable<ReagentUnit>` and `IComparable<ReagentUnit>` here.

Reply:
Done